docs/current_docs/quickstart/conclusion.mdx

```diff
@@ -10,7 +10,7 @@ title: "Conclusion"
 
 This quickstart has walked you through the process of Daggerizing an example application. It has taught you key Dagger concepts and introduced you to the Dagger API, the Dagger CLI, Dagger Functions, Dagger Cloud, and the Daggerverse.
 
-Lastly, you should now have a [Dagger Cloud](https://dagger.cloud/) account, which helps you visualize your pipelines both pre-push and post-push in your CI. This will help you quickly troubleshoot and understand what's happening within any Daggerized pipeline. 
+Lastly, you should now have a [Dagger Cloud](https://dagger.cloud/) account, which helps you visualize your pipelines. This will help you quickly troubleshoot and understand what's happening within any Daggerized pipeline. 
```

Comment:
@vikram-dagger updated here, but I probably updated the PR wrong? Let me know if this is ok. I see your concern with calling out Dagger Cloud here, but I think it is an ok gentle reminder since we want to include more Dagger Cloud tips within the quickstart soon